packages/manager/apps/pci-public-ip/src/api/data/project.ts

```diff
@@ -12,9 +12,9 @@ export interface Project {
 }
 
 export const getProject = async (projectId: string): Promise<Project> => {
-  const response = await v6.get(`/cloud/project/${projectId}`);
+  const { data } = await v6.get(`/cloud/project/${projectId}`);
```

Comment:
Type your response instead of cast it   

`
const { data } = await v6.get<Project>('/cloud/project/${projectId}');
return data;
 `
 
instead of 

`
const { data } = await v6.get('/cloud/project/${projectId}');
return data as Project;
 `